docs/security.rst

```diff
@@ -116,6 +116,95 @@ This process enables HTTPS, but still defaults to HTTP so |sr| instances can sti
 - Do a rolling bounce of the cluster
 
 
+.. _schema_registry_basic_http_auth
+
+Configuring the REST API for Basic HTTP Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|sr| can be configured to require users to authenticate using a username and password via the Basic HTTP authentication mechanism.
+
+.. note:: If you're using Basic authentication, we recommended that you
+          :ref:`configure |sr| to use HTTPS for secure communication <schema_registry_http_https>`,
```

Comment:
I'm not sure whether it's possible to resolve `|sr|` in a reference like this -- had trouble resolving abbreviated names when rendering these updates. Confirmation from someone more familiar with RST would be great. (I checked that the rest of the formatting is as expected when rendered, so that should all be fine.)

Reply:
This is expected: tokens get resolved by the main build from the `docs` repo, so they won't render correctly in the KSQL docs build.

Reply:
@vcrfxia @JimGalasyn they don't render if they are used in cross references or [these other examples](https://github.com/confluentinc/docs#limitations). so in this case, just use the hard-coded `Schema Registry`.